### PR TITLE
Add template cleanup workflow

### PR DESCRIPTION
**Pull request by tiqwab**
Added a template cleanup workflow as discussed in https://github.com/fregante/browser-extension-template/pull/79.

The workflow is almost same as https://github.com/JetBrains/intellij-platform-plugin-template/blob/main/.github/workflows/template-cleanup.yml. It only deletes the below files for now:

- `.github/funding.yml`
- `.github/workflows/template-cleanup.yml` (workflow itself)

### CODE REVIEW

File: .github/workflows/template-cleanup.yml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+          # Remove files
+          rm -f \
+            .github/funding.yml \
+            .github/workflows/template-cleanup.yml
+
+      # Commit modified files
+      - name: Commit files
+        run: |
+          git config --local user.email "action@github.com"
+          git config --local user.name "GitHub Action"
```
Comment (fregante) [inline .github/workflows/template-cleanup.yml]: https://github.com/fregante/setup-git-user or any action that just commits all in 2 lines

**Comment by tiqwab** [comment]
Added changes:

- Configure git user and email as bot by `fregante/setup-git-user`
- Stop using `ad-m/github-push-action` because just `git push` is enough in this case

**Comment by fregante** [comment]
We just need something like

```YML
    - uses: stefanzweifel/git-auto-commit-action@v4
      with:
        commit_message: Apply php-cs-fixer changes

**Comment by fregante** [comment]
It works, thanks!

Screen Shot 8

Screen Shot 9

Screen Shot 10

**Pull request merged**